package.json

```diff
@@ -50,6 +48,7 @@
     "mocha": "2.4.5",
     "mocha-eslint": "2.0.2",
     "mocha-jsdom": "1.1.0",
+    "mz": "2.4.0",
```

Comment:
Mainly it's what I was used to, mz seemed nice as a one-stop-shop for all node APIs. But it looks like fs-promise supports using fs-extra to make a proper copy function so maybe I'll switch to that.